src/services/infra/InfraService.ts

```diff
@@ -327,35 +320,28 @@ export default class InfraService {
       }
 
       if (newLaunchParams.redirectionDomainSource) {
-        message.redirectionDomain = [
-          {
-            source: newLaunchParams.redirectionDomainSource,
-            target: primaryDomainTarget,
-            type: RedirectionTypes.A,
-          },
-        ]
+        const redirectionDomainObject = {
+          source: newLaunchParams.primaryDomainSource,
+          target: REDIRECTION_SERVER_IP,
+          type: RedirectionTypes.A,
+        }
+        message.redirectionDomain = [redirectionDomainObject]
+        newLaunchParams.redirectionDomain = redirectionDomainObject
       }
 
       this.queueService.sendMessage(message)
 
       return ok(newLaunchParams)
     } catch (error) {
-      logger.error(`Failed to create '${repoName}' site on Isomer: ${error}`)
-      // requester email is guaranteed to exist as currently these are Isomer users
-      this.sendRetryToIsomerAdmin(requestor.email!, repoName)
-
-      throw error
+      return err(
+        new AmplifyError(
+          `Failed to create '${repoName}' site on Isomer: ${error}`,
+          repoName
```

Comment:
loses original stack trace on `error`, which kinda defeats the purpose @__@ maybe should try to log or smth

Reply:
Hmm correct me if I am wrong ya, but we are still appending that info to the error message in the object `amplifyError` no? Am I missing something here?

Reply:
oops, my bad, didn't see the `error`